modules/nf-core/falco/main.nf

```diff
@@ -0,0 +1,59 @@
+process FALCO {
+    tag "$meta.id"
+    label 'process_medium'
+
+
+    conda (params.enable_conda ? "bioconda::falco=1.2.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/falco:1.2.1--h867801b_3':
+        'quay.io/biocontainers/falco:1.2.1--h867801b_3' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*.html"), emit: html
+    tuple val(meta), path("*.txt") , emit: txt
+    path  "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    if (meta.single_end) {
+        """
+        [ ! -f  ${prefix}.fastq.gz ] && ln -s $reads ${prefix}.fastq.gz
```

Comment:
What is this doing?

Reply:
Hi, this is copy pasted by fastqc module. Since Falco is a C replacement of fastQC I just followed the same code

Reply:
Can you check all of this linking etc. actually makes sense? It seems to me like `falco` can take an arbitrary number of sequence files to parse. I get that it's copy paste from `fastqc`, but that doesn't mean we can't make it better 😉

Reply:
Nevermind, the linking and separation of SE and PE is done to ensure consistent naming across runs